### ISSUE TITLE
[REQUEST] Live proxy update

### ISSUE THREAD

**Issue by e393** [openbullet/OpenBullet2#273]
1. Have an option to load proxy(s) while job(s) is running so it uses the updated proxy list instead of having to stop the job then load the list and then restart the job. (Similar to live config editing on OB1)

Hella coinvent if scraping for new proxies while job is running all you have to do is load the new proxy list or add it to the group(s) so job uses the new added proxies....

**Comment by openbullet** [comment]
This feature is already implemented, it automatically triggers when the proxies are reloaded (a.k.a. by default when all proxies are banned, but you can manually set it to reload every X minutes as well). It will fetch new proxies from the proxy sources you added, whether they are from the db, a file or a remote API. If you want I can add a button to manually refresh the proxies at any point in time from the job.

**Comment by e393** [comment]
yes i've tried the reload option and i remeber getting an error when that happened because i dont keep my proxies list locally, i load them to the DB and delete the file. But you said they should reload either way, but they never did, might have to go back and try that again to confirm. but the manual button would be nice too.

**Comment by hbibmendas** [comment]
jvjbjhbbynjb
